tests/SpecTests/Operation.php

```diff
@@ -562,9 +556,7 @@ private function executeForGridFSBucket(Bucket $bucket, Context $context)
     /**
      * Executes the session operation and return its result.
      *
-     * @param Session            $session
-     * @param FunctionalTestCase $test
-     * @param Context            $context Execution context
+     * @param Context $context Execution context
```

Comment:
Feel free to remove `* @param Context $context Execution context` everywhere if you don't think it adds value.

`@return mixed` can go as well if you don't think it's helpful.

Reply:
Removed this and a number of other superfluous type annotations.